src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/SetAzurePolicyAssignment.cs

```diff
@@ -131,9 +122,9 @@ private JToken GetResource(string resourceId, string apiVersion)
             var policyAssignmentObject = new PolicyAssignment
             {
                 Name = this.Name ?? ResourceIdUtility.GetResourceName(this.Id),
-                Sku = this.Sku != null
-                    ? this.Sku.ToDictionary(addValueLayer: false).ToJson().FromJson<PolicySku>()
-                    : (resource.Sku == null ? new PolicySku { Name = "A0", Tier = "Free" } : resource.Sku.ToJson().FromJson<PolicySku>()),
+                //Sku = this.Sku != null
```

Comment:
Since SKU still exists in the API for now I think this should persist it if the user is explicitly providing it in the cmdlet.  It should not include it if they did not include it

Reply:
Sure, I can put it back. The reason I removed it is because I wanted to mark the parameter [Obsolete]. But doing so means any reference to the parameter causes a build break. So we have to choose between obsoleting it and keeping it. I guess you still want to keep it? Also, I guess you are still OK with throwing it away for New-AzureRmPolicyAssignment?

Reply:
ah, if you are moving towards making it obsolete here then its fine.  Not sure how long that process takes (i.e. how many PS releases the obsolete tag is around until the parameter can be removed)

Reply:
OK, I'll add it back to the PR and we can cover that in the design meeting as well. I don't know what the obsoleting process looks like either.